tests/validators/answers/test_option_answer_validator.py

```diff
@@ -227,13 +256,21 @@ def test_dynamic_options_transform_with_invalid_value_rule():
     )
     validator.validate_dynamic_options()
 
-    expected_error = {
+    option_label_from_value_do_not_match_error = {
+        "message": validator.DYNAMIC_OPTIONS_SOURCE_IDENTIFIER_AND_OPTION_LABEL_FROM_VALUE_MISMATCH,
+        "answer_id": "answer",
+    }
+
+    answer_reference_invalid_error = {
         "message": ValueSourceValidator.ANSWER_REFERENCE_INVALID,
         "identifier": "non-existing-answer",
         "origin_id": "answer",
     }
 
-    assert validator.errors == [expected_error]
+    assert validator.errors == [
```

Comment:
Because I have an answer_id which doesn't exists, it will not only trigger the mismatch error but the answer_reference_invalid error which looks for existence in a schema